nci/workspace_cleaner.rb

```diff
@@ -81,9 +82,9 @@ def cleanup?(workspace)
         puts '  ws-cleanup => delete'
         return true
       end
-      # Never delete mgmt workspaces. Too dangerous as they are
-      # persistent.
-      if workspace.include?('mgmt_')
+       # Never delete current or future (series) mgmt workspaces.
+      # Too dangerous as they are persistent.
+      if workspace.include?('mgmt_#{NCI.current_series}' or 'mgmt_#{NCI.future_series}')
```

Comment:
That `or` not valid code. It's also using string literals meaning the variables won't get expanded.

Reply:
apologies and thanks for your patience.  obviously it's not my strongest suite. ;)